swap_meet/vendor.py

```diff
@@ -1,2 +1,76 @@
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+            
+
+    def add(self, add_new_item):
+        self.inventory.append(add_new_item)
+        return (add_new_item)
+
+    def remove(self, remove_item):
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return (remove_item)
+        else:
+            return False
+
+    def get_by_category(self, category_name):
+        item_list = []
+        for inventory in self.inventory:
+            if inventory.category == category_name:
+                item_list.append(inventory)
+        return item_list
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item  in self.inventory and their_item in vendor.inventory:
+            self.inventory.append(their_item)
+            vendor.inventory.remove(their_item)
+            vendor.inventory.append(my_item)
+            self.inventory.remove(my_item)
+            return True
+        if my_item not in self.inventory or their_item not in vendor.inventory:
+            return False
```

Comment:
*Minor note*:
I don't think this `if ` is needed

```suggestion
        return False
```